include/mbedtls/ctr_drbg.h

```diff
@@ -35,7 +35,11 @@
 #define MBEDTLS_ERR_CTR_DRBG_FILE_IO_ERROR                -0x003A  /**< Read/write error in file. */
 
 #define MBEDTLS_CTR_DRBG_BLOCKSIZE          16      /**< Block size used by the cipher                  */
+
+#if !defined(MBEDTLS_CTR_DRBG_KEYSIZE)
```

Comment:
Why do we need this guard here?

Reply:
Hi, this should allow one to do "-DMBEDTLS_CTR_DRBG_KEYSIZE=16" on CFLAGS and get DRBG with AES-128 that way. MBEDTLS_CTR_DRBG_KEYBITS and MBEDTLS_CTR_DRBG_SEEDLEN depend specifically on this variable. 
There is a requirement for AES-128 based DRBG on some lightweight targets that only have that in their hardware. We are of course missing tests for this case.